cli/src/main/java/org/jboss/as/cli/impl/CliConfigImpl.java

```diff
@@ -94,6 +94,7 @@
     static CliConfig newBootConfig() throws CliInitializationException {
         CliConfigImpl config = new CliConfigImpl();
         config.validateOperationRequests = false;
+        config.echoCommand = true;
```

Comment:
is this change required to support the new `--cli-script`?

Reply:
No, not required. The error reporting is now better.